ext/opcache/config.m4

```diff
@@ -81,16 +82,13 @@ if test "$PHP_OPCACHE" != "no"; then
         ;;
      esac
 
-    AS_IF([test x"$with_capstone" = "xyes"],[
-      PKG_CHECK_MODULES([CAPSTONE],[capstone >= 3.0.0],[
-        AC_DEFINE([HAVE_CAPSTONE], [1], [Capstone is available])
+    AS_VAR_IF([PHP_CAPSTONE], [yes],
+      [PKG_CHECK_MODULES([CAPSTONE], [capstone >= 3.0.0], [
+        AC_DEFINE([HAVE_CAPSTONE], [1], [Define to 1 if Capstone is available.])
         PHP_EVAL_LIBLINE([$CAPSTONE_LIBS], [OPCACHE_SHARED_LIBADD])
         PHP_EVAL_INCLINE([$CAPSTONE_CFLAGS])
         ZEND_JIT_SRC="$ZEND_JIT_SRC jit/ir/ir_disasm.c"
-      ],[
-        AC_MSG_ERROR([capstone >= 3.0 required but not found])
-      ])
-    ])
```

Comment:
What is the reason to remove this?

Reply:
Default PKG_CHECK_MODULES error message is thrown if the 4th argument is empty:

```
checking whether to enable JIT... yes
checking whether to support opcache JIT dissasembly through Capstone... yes
checking for capstone >= 3.0.0... no
configure: error: Package requirements (capstone >= 3.0.0) were not met:

Package 'capstone' not found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables CAPSTONE_CFLAGS
and CAPSTONE_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
```

otherwise if the 4th argument is added, the error message is like this:

```
checking whether to enable JIT... yes
checking for capstone >= 3.0.0... no
configure: error: capstone >= 3.0 required but not found
```